### PR TITLE
Update step validating sorting order of elements

### PR DESCRIPTION
**Pull request by Tatsiana-Luchonak**

### CODE REVIEW

File: vividus-tests/src/main/resources/story/integration/ElementStepsTests.story [hunk 1]
```diff
@@ -194,5 +194,10 @@ Given I am on a page with the URL '${vividus-test-site-url}'
 Then number of VISIBLE elements found by `tagName(img):a` is = `1`
 
 Scenario: Step verification Then elements located `$locator` are sorted by text in $sortingOrder order
+!-- Deprecated
 Given I am on a page with the URL '${vividus-test-site-url}/sortedListOfElement.html'
 Then elements located `By.xpath(//div//h3)` are sorted by text in ASCENDING order
+
+Scenario: Step verification Then elements located `$locator` are sorted by text in $sortingOrder order
```
Comment (uarlouski) [inline vividus-tests/src/main/resources/story/integration/ElementStepsTests.story:201]: located by

File: vividus/src/main/resources/steps/defaults/composite.steps [hunk 2]
```diff
@@ -8,3 +8,8 @@ Composite: Given I initialize the $scopes variable `$variableName` using templat
 Priority: 1
 !-- WARNING: The step "Given I initialize the $scopes variable `$variableName` using template `$templatePath` with parameters:$templateParameters" is deprecated and will be removed in VIVIDUS 0.6.0
 Given I initialize <scopes> variable `<variableName>` using template `<templatePath>` with parameters:<templateParameters>
+
+
+Composite: Then elements located `$locator` are sorted by text in $sortingOrder order
+!-- WARNING: The step "Then elements located `$locator` are sorted by text in $sortingOrder order" is deprecated and will be removed in VIVIDUS 0.6.0.
+Then elements located by `$locator` are sorted by text in $sortingOrder order
```
Comment (uarlouski) [inline vividus/src/main/resources/steps/defaults/composite.steps:15]: wrong refs

File: docs/modules/plugins/partials/generic-ui-steps.adoc [hunk 3]
```diff
@@ -291,6 +291,16 @@ element with not empty text, otherwise the step fails.
...
+[WARNING]
+====
+The step is deprecated and will be removed in VIVIDUS 0.6.0.
+The replacement pattern is:
+[source,gherkin]
+----
+Then elements located by `$locator` are sorted by text in $sortingOrder order
+----
+====
+
```
Comment (uarlouski) [inline docs/modules/plugins/partials/generic-ui-steps.adoc:303]: .Deprecated syntax
[source,gherkin]
----
 Then elements located `$locator` are sorted by text in $sortingOrder order
 ----

[source,gherkin]
----
 Then elements located by `$locator` are sorted by text in $sortingOrder order
 ----

File: docs/modules/plugins/partials/generic-ui-steps.adoc [hunk 4]
```diff
@@ -324,7 +330,7 @@ Then elements located `$locator` are sorted by text in $sortingOrder order
 ----
 Given I am on a page with the URL 'https://grocery.by'
 When I click on element located by `id(a-z)`
-Then elements located `By.xpath(//span[@id='item-to-purchase'])` are sorted by text in ascending order
+Then elements located by `By.xpath(//span[@id='item-to-purchase'])` are sorted by text in ascending order
```
Comment (valfirst) [inline docs/modules/plugins/partials/generic-ui-steps.adoc:333]:
```suggestion
Then elements located by `xpath(//span[@id='item-to-purchase'])` are sorted by text in ascending order
```

File: vividus-tests/src/main/resources/story/integration/ElementStepsTests.story [hunk 5]
```diff
@@ -194,5 +194,10 @@ Given I am on a page with the URL '${vividus-test-site-url}'
 Then number of VISIBLE elements found by `tagName(img):a` is = `1`
 
 Scenario: Step verification Then elements located `$locator` are sorted by text in $sortingOrder order
+!-- Deprecated
 Given I am on a page with the URL '${vividus-test-site-url}/sortedListOfElement.html'
 Then elements located `By.xpath(//div//h3)` are sorted by text in ASCENDING order
+
+Scenario: Step verification Then elements located by `$locator` are sorted by text in $sortingOrder order
+Given I am on a page with the URL '${vividus-test-site-url}/sortedListOfElement.html'
```
Comment (valfirst) [inline vividus-tests/src/main/resources/story/integration/ElementStepsTests.story:202]: I guess this step is redundant

**Review by valfirst — CHANGES_REQUESTED** [review]

**Review by uarlouski — APPROVED** [review]

**Review by valfirst — APPROVED** [review]

File: vividus-tests/src/main/resources/story/integration/ElementStepsTests.story [hunk 6]
```diff
@@ -194,5 +194,9 @@ Given I am on a page with the URL '${vividus-test-site-url}'
 Then number of VISIBLE elements found by `tagName(img):a` is = `1`
 
 Scenario: Step verification Then elements located `$locator` are sorted by text in $sortingOrder order
+!-- Deprecated
 Given I am on a page with the URL '${vividus-test-site-url}/sortedListOfElement.html'
 Then elements located `By.xpath(//div//h3)` are sorted by text in ASCENDING order
+
+Scenario: Step verification Then elements located by `$locator` are sorted by text in $sortingOrder order
+Then elements located by `xpath(//div//h3)` are sorted by text in ASCENDING order
```
Comment (ikalinin1) [inline vividus-tests/src/main/resources/story/integration/ElementStepsTests.story:202]:
```suggestion
Then elements located by `tagName(h3)` are sorted by text in ASCENDING order
```

Comment (Tatsiana-Luchonak) [reply vividus-tests/src/main/resources/story/integration/ElementStepsTests.story:202]: It is the same scenario that was before deprecation of step, this part is copied from already existed

Comment (valfirst) [inline vividus-tests/src/main/resources/story/integration/ElementStepsTests.story:202]:
```suggestion
Then elements located `By.tagName(h3)` are sorted by text in ASCENDING order

Scenario: Step verification Then elements located by `$locator` are sorted by text in $sortingOrder order
Then elements located by `tagName(h3)` are sorted by text in ASCENDING order
```

**Pull request merged**